.github/workflows/chart-lint.yml

```diff
@@ -0,0 +1,72 @@
+name: Install and Lint Charts
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+    - 'metrics/prometheus/operator/values.yaml'
+    - 'metrics/prometheus/operator/Chart.yaml'
+
+env:
+  CHART_VERSION: $(yq eval '.version' prometheus/operator/Chart.yaml)
+  CHART_NAME: prometheus-operator-coralogix
+  ARTIFACTORY_URL: https://cgx.jfrog.io/artifactory/
+  ARTIFACTORY_USERNAME: integrations-actions
+
+jobs:
+  install-lint-test:
+    strategy:
+      matrix:
+        suite: [fluent-bit, fluentd, prometheus-operator, prometheus-agent, otel-agent, otel-infra]
+        include:
+          - suite: fluent-bit
+            dependency: fluent-bit
+            dependency-chart-repo: https://fluent.github.io/helm-charts
+            chart-path: logs/fluent-bit/k8s-helm/coralogix
+            install-prom-operator-crd: false
+          - suite: fluentd
+            dependency: fluent-bit
+            dependency-chart-repo: https://fluent.github.io/helm-charts
+            chart-path: logs/fluentd/k8s-helm/coralogix
+            install-prom-operator-crd: false
+          - suite: prometheus-operator
+            dependency: kube-prometheus-stack
+            dependency-chart-repo: https://prometheus-community.github.io/helm-charts
+            chart-path: metrics/prometheus/operator
+            install-prom-operator-crd: true
+          - suite: prometheus-agent
+            dependency: ""
+            dependency-chart-repo: ""
+            chart-path: "metrics/prometheus-agent"
+            install-prom-operator-crd: true
+          - suite: otel-agent
```

Comment:
hey, unfortunately my priorities shifted before I could finish these, sorry for not commenting anything here for so long 😬 

It would be awesome if someone could take this over